### PR TITLE
Pokeball throwing based on IV and Catching Probability instead of CP.

### PR DESCRIPTION
**Pull request by DeadElle**
• UltraBallBelowCatchProbability set to 0.25 to not use them when it's unnecessary.
• UseBerryBelowCatchProbability to 0.2 due to continuos unnecessary use for low pokemons.
• Enabled transfer of Spearow, Zubat and Doduo for exp maximizing.
• PokeBalls throwing based on IV and catching probability % instead of CP.

### CODE REVIEW

File: PoGo.PokeMobBot.Logic/Settings.cs [hunk 1]
```diff
@@ -155,7 +155,7 @@ public class GlobalSettings
         public bool KeepPokemonsThatCanEvolve = false;
 
         //evolve
-        public bool EvolveAllPokemonWithEnoughCandy = false;
+        public bool EvolveAllPokemonWithEnoughCandy = true;
```
Comment (DurtyFree) [inline PoGo.PokeMobBot.Logic/Settings.cs]: Please change back to default "false"

File: PoGo.PokeMobBot.Logic/Settings.cs [hunk 2]
```diff
@@ -284,7 +281,7 @@ public class GlobalSettings
             //PokemonId.Bulbasaur,
             //PokemonId.Charmander,
             //PokemonId.Squirtle,
-            PokemonId.Rattata
+            PokemonId.Rattata,
```
Comment (DurtyFree) [inline PoGo.PokeMobBot.Logic/Settings.cs]: Please take a look at our actual pokemontoevolve defaults

**Comment by DurtyFree** [comment]
Please take a look at my comments and fix the conflicts thx:) @DeadElle

File: PoGo.PokeMobBot.Logic/Settings.cs [hunk 3]
```diff
@@ -297,14 +294,14 @@ public class GlobalSettings
...
+            PokemonId.Spearow,
             //PokemonId.Ekans,
-            //PokemonId.Zubat,
+            PokemonId.Zubat,
             //PokemonId.Paras,
             //PokemonId.Venonat,
             //PokemonId.Psyduck,
             //PokemonId.Slowpoke,
-            //PokemonId.Doduo,
+            PokemonId.Doduo
```
Comment (DeadElle) [inline PoGo.PokeMobBot.Logic/Settings.cs]: Actually Doduo is the last one of the list

**Comment by DurtyFree** [comment]
Can you please fix the merge conflicts?:)
@DeadElle

**Comment by DeadElle** [comment]
@DurtyFree done :)

**Comment by DurtyFree** [comment]
Nice thx:)

**Pull request merged**